### PR TITLE
fix(playground): _SNOWFLAKE_CONNECTION needs to be defined

### PR DESCRIPTION
**Pull request by lukapeschke**
Signed-off-by: Luka Peschke <luka.peschke@toucantoco.com>

### CODE REVIEW

File: server/playground.py [hunk 1]
```diff
@@ -359,7 +359,7 @@ async def handle_mongo_translated_backend_request():
             client_session_keep_alive=True,
         )
     except Exception:
-        _SNOWFLAKE_CONNECTION = None
+        pass
```
Comment (davinov) [inline server/playground.py:362]: Maybe we can just log that the connection failed?

Comment (lukapeschke) [reply server/playground.py:362]: Sure! Is a simple `print` okay for you ?

Comment (davinov) [reply server/playground.py:362]: just to make sure it's not silent in the console

Comment (lukapeschke) [reply server/playground.py]: Added a print and checked, it's visible in the console :slightly_smiling_face:

**Review by davinov — APPROVED** [review]

**Pull request merged**